assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -144,25 +144,32 @@ def compute_explanations(model_wrapper, data, categorical_features, target_colum
     :type target_column: string
     :param task_type: the task type (regression or classification) of the resulting model
     :type task_type: string
-    :return: explanation scores for the input data
-    :rtype: list[float]
+    :return: global explanation scores for the input data and local explanation scores for the input data
+    :rtype: list[float], list[float]
     """
     # Create the RAI Insights object, use baseline as train and test data
     feature_metadata = FeatureMetadata(categorical_features=categorical_features, dropped_features=[])
     rai_i: RAIInsights = RAIInsights(
         model_wrapper, data, data, target_column, task_type, feature_metadata=feature_metadata
     )
     log_time_and_message("Created RAIInsights")
-    # Add the global explanations using batching to allow for larger input data sizes
     rai_i.explainer.add()
     evaluation_data = data.drop([target_column], axis=1)
-    log_time_and_message("Requesting explanations")
-    explanationData = rai_i.explainer.request_explanations(local=False, data=evaluation_data)
-    return explanationData.precomputedExplanations.globalFeatureImportance['scores']
+    # Add the global explanations using batching to allow for larger input data sizes
```

Comment:
As a general rule it's always best to avoid tuple return types (since they're super prone to error and implies the underlying method is overloaded with functionality)

I wonder if it'd be easy to split this into a get_local_explanation/get_global_explanation?